travel.py

```diff
@@ -221,7 +238,11 @@ def flights(tenant, username):
                 for key in booked_flights:
                     rows.append(flights.get(key).content_as[dict])
                 print(rows)
-                respjson = jsonify({"data": rows})
+
+                querytype = "KV get - scoped to " + scope.name + \
```

Comment:
I love the new output from the backend!

Python's string.format might be more idiomatic/readable here btw? Forget details, but something like:
`"scoped to {name}.user: for {num} bookings in document".format(name=..., num=...)`

Reply:
Roger roger! 🤖